CHANGELOG.rst

```diff
@@ -18,7 +18,11 @@ Unreleased
 [0.0.8] - 2018-03-01
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-* Nothing
+* Added "utilities.py", which houses methods for working with BlockCompletion
+  data.
+* Add model method for superlative “last completed block” - for site awareness 
```

Comment:
This sentence makes no sense and is also not true - there is no added method on the BlockCompletion model in this PR.

Reply:
@iloveagent57 I added it from Greg's PR, which bumps the version number:
https://github.com/edx/completion/pull/9

My understanding is that release 0.0.8 combines Greg and my work and that his PR completes the first half of the release process.